docs/Makefile

```diff
@@ -2,13 +2,20 @@
 
 IMAGE = grafana/docs-base:latest
 
-docs:
-	docker pull ${IMAGE}
+docs-generate-frontend:
+	yarn install --pure-lockfile --no-progress
+	../scripts/ci-reference-docs-build.sh
```

Comment:
Can we please avoid using a CI script in Makefile? I think it's a bad idea to ask people to use CI scripts locally, as those scripts might be changed to do something that shouldn't happen locally, exactly for the reason that they're meant for CI.

I would instead just extract the `yarn` commands from that script.

Reply:
![](https://media.giphy.com/media/KAj8a8FLY0ufNieMkV/giphy.gif)